test/blackbox-tests/test-cases/mdx-stanza/paths.t

```diff
@@ -0,0 +1,160 @@
+Absolute paths cause an error.
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.0)
+  > (using mdx 0.2)
+  > EOF
+  $ cat > dune << EOF
+  > (mdx)
+  > EOF
+
+  $ cat > README.md << 'EOF'
+  > ```ocaml file=/etc/passwd
+  > ```
+  > EOF
+
+  $ dune runtest
+  Internal error, please report upstream including the contents of _build/log.
+  Description:
+    ("Local.relative: received absolute path",
+    { t = "."; path = "/etc/passwd" })
+  Raised at Stdune__Code_error.raise in file "otherlibs/stdune/code_error.ml",
```

Comment:
Do you have a fix ready? Otherwise I suggest you redact the stacktrace. We don't want this test to be updated on unrelated commits.

Reply:
I do but what's the technique to redact the stacktrace?

Reply:
Perhaps pipe it to awk?

```
.. | awk '/Internal error/,/Raised at/'
```